appveyor.yml

```diff
@@ -1,8 +1,8 @@
-version: 5.0.0.{build}
+version: 6.0.0.{build}
```

Comment:
This can stay as 5.0.0, we haven't release 5.0.0 of Multitenant yet (the compatibility for v5 has already started on develop).

Reply:
Addressed.